src/test/scala/SystemTests.scala

```diff
@@ -36,48 +26,52 @@ trait SystemTests extends AnyFunSuite {
 
   val testResults: mutable.ArrayBuffer[(String, TestResult)] = mutable.ArrayBuffer()
 
-  def runTests(programs: Array[String], path: String, name: String, conf: TestConfig): Unit = {
+  private val testPath = "./src/test/"
+
```

Comment:
Can we set   `Logger.setLevel(LogLevel.WARN)`, printing the globals, proc addresses and then the same list of translation stage log messages for every test really isn't useful and makes it harder to see the failing/passing tests.

Reply:
I just replaced most of the Logger.info calls with Logger.debug, only keeping ones to print the very basic test progress.